src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+public class Parser {
+
+    /**
+     * Parses the command into a simpler form.
+     * @param command A String given by the user.
+     * @return An array of strings to be handled by Duke.
+     * @throws DukeException
+     */
+    public static String[] parse(String command) throws DukeException {
+        String[] splitCommands = command.split(" ", 2);
+        switch (splitCommands[0]) {
+        case "list":
+        case "sort":
+        case "bye":

```

Comment:
You may want to use enums to capture the different strings that pertain to each command, in order to facilitate re-use and consistency across files that use these names (e.g Duke.java).

Alternatively, you may want to represent commands using classes for further abstraction, but this would require significantly more effort.